src/constants.rs

```diff
@@ -167,6 +167,11 @@ pub const DC_MSG_ID_LAST_SPECIAL: u32 = 9;
 
 /// String that indicates that something is left out or truncated.
 pub const DC_ELLIPSIS: &str = "[...]";
+// how many lines desktop can display when fullscreen (fullscreen at zoomlevel 1x)
+// (taken from "subjective" testing what looks ok)
+pub const DC_DESIRED_TEXT_LINES: usize = 38;
```

Comment:
why focus on desktop here?

Reply:
because android and iOS likely have smaller screens. Desktop can have the biggest screen space compared to mobile.

Reply:
also BTW this pr is meant for iOS, desktop is just the platform I chose for figuring out the limits.